src/components/Dashboard/FilingHistoryList.vue

```diff
@@ -168,8 +168,11 @@
     <!-- No Results Message -->
     <v-card class="no-results" flat v-if="!filedItems.length">
       <v-card-text>
-        <div class="no-results__title">You have no filing history</div>
-        <div class="no-results__subtitle">Your completed filings and transactions will appear here</div>
+        <div class="no-results__subtitle"  v-if="nrNumber">Complete your filing to display</div>
+        <div v-else>
```

Comment:
This creates another layer of divs. Sometimes divs have padding/spacing -- did you test that this looks OK (no filing history)?
One solution is to use the <template> element.

Reply:
Changed it to template